lwatools/vis_modeling/vis_model_fitting.py

```diff
@@ -46,59 +35,81 @@ def ls_cost(params, u, v, vis, resid=point_residual_abs):
     return np.dot(r,r)
 
 
+def fit_model_to_vis(bl, freqs, vis, target_bin, residual_function, l_init, m_init,
```

Comment:
maybe `fit_point_source_model_to_vis` to clarify but yea our names could all use work ha

Reply:
You can pass in whatever residual function you want so really you could fit whatever model you want as long as it has two parameters (l,m)